lib/deep-equal.test.js

```diff
@@ -830,4 +812,76 @@ describe("deepEqual", function() {
             assert.isFalse(checkDeep);
         });
     });
+
+    describe("arrays", function() {
+        function arrayOf(Klass) {
```

Comment:
This would be better named `instantiate` ... which it what it does.
```suggestion
        function instantiate(Klass) {
```